rewrite-java-17/src/main/java/org/openrewrite/java/isolated/ReloadableJava17JavadocVisitor.java

```diff
@@ -668,7 +668,7 @@ public Tree visitProvides(ProvidesTree node, List<Javadoc> body) {
                         randomId(),
                         qualifier == null ? Space.EMPTY : qualifier.getPrefix(),
                         Markers.EMPTY,
-                        qualifier == null ? null : JRightPadded.build(qualifier.withPrefix(Space.EMPTY)),
+                        JRightPadded.build(qualifier == null ? null : qualifier.withPrefix(Space.EMPTY)),
```

Comment:
This mimics what we do for the Java parser: there we also unconditionally create a JRightPadded with a nullable containing element.